Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1331,6 +1335,17 @@ class BrowserViewController: UIViewController {
                 // for reasons above about URL spoofing risk.
                 navigateInTab(tab: tab)
             }
+            
+            // Rewards reporting
```

Comment:
I moved it directly under URL KVO and removed `mediaSite` path, now we report page load on all internal page navigations, previously it was only for youtube, twitch etc..

This was also moved because `updateURLBar` is called from few places, and we want to avoid duplicate calls for `reportPageLoad`